run_serial_workflow.py

```diff
@@ -127,8 +134,44 @@
 
             # Update the record
             record['valid'] = True
+            all_ligands.append(record)
+
+            # begin of swap cyano for COO
+            coo_ligand = ligand.swap_cyano_with_COO()
+            coo_record = {"anchor_type": coo_ligand.anchor_type, "xyz": coo_ligand.xyz,
+                      "anchor_atoms": coo_ligand.anchor_atoms, "valid": False}
+
+            # Try constrained optimization on the ligand
+            try:
```

Comment:
Constrained optimization has something to do with the conversion, so could you move this to within `swap_cyano_with_COO()`?